lib/utils.js

```diff
@@ -0,0 +1,12 @@
+import * as _ from 'underscore';
+
+function areObjectsEmpty(a, b) {
+    return (
+        typeof a === 'object'
+        && typeof b === 'object'
+        && _.values(a).length === 0
```

Comment:
What's the difference between this and using `_. isEmpty_()` or `_.size() === 0`?

Reply:
Initial implementation was Object.values(a).length, linter barked at me I should use underscore, no other reason. I will change it to `_.isEmpty`